### ISSUE TITLE
Delay resizing windows while dragging a Chrome tab

### ISSUE THREAD

**Issue by dustin-cowles** [ianyh/Amethyst#1292]
**Is your feature request related to a problem? Please describe.**
When trying to drag a Chrome browser tab from one window to another, Chrome spawns a new window under the cursor as soon as the tab is dragged out of a window. This causes amethyst to immediately start resizing existing windows to make space for the (temporary) new window. This makes it very difficult to move the tab into the other window.

The issue is easiest to understand when you have start with two or more Chrome windows in a Tall or similar layout.

**Describe the solution you'd like**
When dragging a tab out of one window, it would be nice if Amethyst detected this and did not evaluate/resize any windows until the tab was released either as a new window or into another existing one.

**Describe alternatives you've considered**
It is possible to right click a tab and move to another window, but I often have many windows on multiple desktops and some minimized so it can be difficult to identify the correct window from the menu.

**Additional context**
Issue observed with Chrome (and likely most Chrome/ium-based browsers) but does not occur with Firefox or Safari due to those browsers not spawning a new window until the tab is dropped/released.

Video: https://youtu.be/BTH0ON9ziJQ

**Comment by vanakema** [comment]
Yeah I've found this to be a pretty annoying behavior too. I'm thinking it'd be nice if Amethyst waited till the mouse button was not held down to move things around. I think it's a reasonable assumption that if a user is in the middle of dragging something that maybe it's not the best time to rearrange things. There could be edge cases I'm not thinking about though
